pySDC/playgrounds/Runge_Kutta/test_order.py

```diff
@@ -0,0 +1,53 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+from pySDC.projects.Resilience.accuracy_check import plot_orders
+
+from pySDC.projects.Resilience.advection import run_advection
+from pySDC.projects.Resilience.vdp import run_vdp
+
+from pySDC.implementations.sweeper_classes.Runge_Kutta import RK1, RK4, MidpointMethod
+
+
+def plot_order(sweeper, prob, dt_list, description=None, ax=None, Tend_fixed=None):
+    if ax is None:
+        fig, ax = plt.subplots(1, 1)
+
+    description = dict() if description is None else description
+    description['sweeper_class'] = sweeper
+    description['sweeper_params'] = {'implicit': True}
+
+    # determine the order
+    plot_orders(ax, [1], True, Tend_fixed=Tend_fixed, custom_description=description, dt_list=dt_list, prob=prob)
+
+    # decorate
+    colors = {
+        RK1: 'blue',
+        MidpointMethod: 'red',
+        RK4: 'orange',
+    }
+    ax.get_lines()[-1].set_color(colors.get(sweeper, 'black'))
+
+    label = f'{sweeper.__name__} - {ax.get_lines()[-1].get_label()[5:]}'
+    ax.get_lines()[-1].set_label(label)
+    ax.legend(frameon=False)
+
```

Comment:
Please add tests here, making sure that the expected order is reached, see e.g. [here](https://github.com/Parallel-in-Time/pySDC/blob/master/pySDC/tutorial/step_1/B_spatial_accuracy_check.py#L47).